src/test/java/com/esendex/java/sdk/integration/messagingservice/SendServiceSingleUnicodeSMSIT.java

```diff
@@ -0,0 +1,37 @@
+package com.esendex.java.sdk.integration.messagingservice;
+
+import com.esendex.java.sdk.BaseTest;
+import esendex.sdk.java.EsendexException;
+import esendex.sdk.java.model.domain.request.SmsMessageRequest;
+import esendex.sdk.java.model.domain.response.MessageResultResponse;
+import esendex.sdk.java.model.types.CharacterSet;
+import esendex.sdk.java.service.MessagingService;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class SendServiceSingleUnicodeSMSIT extends BaseTest {
+
+    private static MessageResultResponse resultResponse;
+
+    @BeforeClass
+    public static void whenSendingASingleMessage() throws EsendexException {
+
+        MessagingService messagingService = getFactory().getMessagingService();
+
+        resultResponse = messagingService.sendMessage(ACCOUNT, new SmsMessageRequest(DESTINATION_NUMBER, "\u1F43", CharacterSet.UNICODE));

```

Comment:
None of the tests prove anything - this is a carbon copy of what exists. RIP. :skull: :hear_no_evil: